src/terra-dev-site/doc/application/how-to.2/use-translated-strings.doc.mdx

```diff
@@ -1,25 +1,26 @@
 # How To Use Translated Strings
 
-Translated strings can be retrieved using the `intl` object provided by `injectIntl` HOC-generator provided by [react-intl](https://github.com/formatjs/react-intl/blob/master/docs/API.md#injectintl).
+Translated strings can be retrieved using the [ApplicationIntlContext](/components/terra-application/application/reference/contexts/application-intl-context). An `ApplicationIntlContext.Provider` is configured and provided by [ApplicationBase](/components/terra-application/application/reference/components/application-base). The strings will be translated based upon the `locale` prop provided to ApplicationBase.
+
+In addition to the ApplicationIntlContext, the components provided by react-intl can be used directly. However, the ApplicationIntlContext can be used by Hooks-based components to access i18n capabilities without having to use the `injectIntl` HOC-generator from react-intl.
```

Comment:
`ApplicationIntlContext`, the components provided by `react-int`

Reply:
Are we trying to pushing this stategy? If so, we should switch the order of these sentances:

The ApplicationIntlContext **is useful** because it can be used by Hooks-based components to access i18n capabilities without having to use the `injectIntl` HOC-generator from react-intl. However, components  provided by react-intl can be used directly.

Reply:
I don't think we're trying to "push" it per se. It's more along the lines of "We use react-intl. You should use react-intl. If for some reason you want/need to use a Context before the react-intl upgrade, you can use the ApplicationIntlContext."

I'm going to refactor this how-to quite a bit to primarily point to the existing I18n docs on terra-ui.com. It's all useful stuff, but I don't want to dupe it here. But I'll make explicit mention here of the ApplicationIntlContext and scenarios where it might be useful.